### DISCUSSION TITLE
Feature Request: On "Search" Show photos being filters out from Low Quality library settings

### DISCUSSION THREAD

**Discussion by ephekt** [photoprism/photoprism#5363] [Ideas]
I ran into this issue and after a few hour of debugging and Reddit grepping, I discovered that by default mobile screenshots and low-quality scored photos are removed from the Search view. I would suggest in this feature/UX improvement request to show, on the Search page, how many items have been filtered out/hidden. Maybe hovering over it could say "Quality Filter setting enabled on Libraries, filtering X out"

That kind of discoverability would be so nice!

**Comment by graciousgrey** [comment]
Those images are displayed in the review section --> https://docs.photoprism.app/user-guide/organize/review/ 
If you want to see them in the normal search you can search for quality:0 --> https://docs.photoprism.app/user-guide/search/filters/#filter-reference

**Comment by ephekt** [comment]
Thank you. You are welcome to close the feature request if you don't feel it would help others. Your options are not related to my feature suggestion; the reason why is if you need to know that the Quality Filter is on, you may not know to search using the filter quality:0. It's a discoverability feature request. Again, fine if this is just one of those things no one brings up and I'm an outlier. Thanks for considering.

Another idea is to maybe show, on the Search header area, a badge count on what needs to be reviewed. I've been using PhotoPrism for almost a year, and I had no idea the Review section existed. It was only this last week I went looking for a specific thing (I'd admittedly usually not look for in PhotoPrism, a screenshot) that I ran into this.

**Comment by graciousgrey** [comment]
We can keep it open for a while to see if this would be interesting to others as well.
